v3/internal/doctor/doctor.go

```diff
@@ -160,8 +161,12 @@ func Run() (err error) {
 			if len(gpu.GraphicsCards) > 1 {
 				prefix = "GPU " + strconv.Itoa(idx+1) + " "
 			}
-			details := fmt.Sprintf("%s (%s) - Driver: %s", card.DeviceInfo.Product.Name, card.DeviceInfo.Vendor.Name, card.DeviceInfo.Driver)
-			systemTabledata = append(systemTabledata, []string{prefix, details})
+			if card.DeviceInfo != nil {
+				details := fmt.Sprintf("%s (%s) - Driver:%s ", card.DeviceInfo.Product.Name, card.DeviceInfo.Vendor.Name, card.DeviceInfo.Driver)
+				systemTabledata = append(systemTabledata, []string{prefix, details})
+				return
```

Comment:
should this be `continue` rather than return?

Reply:
woops sorry. yup.